docs/v1.md

```diff
@@ -50,6 +58,7 @@
 - filterBy(filter: `(channel: Channel) => boolean`): `Channel[]`
 - filterBySend(): `Channel[]`
 - filterByReceive(): `Channel[]`
+- filterByTags(tags: `string[]`): `Channel[]`
```

Comment:
I guess the intent would be to filter channels by tags located in Servers, Operations, Messages...
But is this needed?